ts/DataGrid/Options.ts

```diff
@@ -391,6 +397,33 @@ export interface HeaderEvents {
     click?: ColumnEventCallback;
 }
 
+export interface GroupedHeader {
+    /**
+     * The format of the column header. Use `{id}` to display the column id.
+     */
+    headerFormat?: string;
+    /**
+     * Weather to use HTML to render the cell content. When enabled, other
+     * elements than text can be added to the cell ie. images.
+     */
+    useHTML?: boolean;
+    /**
+     * The custom CSS class name for the table caption.
+     */
+    className?: string;
```

Comment:
![Screenshot 2024-08-19 at 15 56 36](https://github.com/user-attachments/assets/a64f0114-04fc-40a5-9a93-f854cafa3337)
![Screenshot 2024-08-19 at 15 56 44](https://github.com/user-attachments/assets/16e3ec9a-1eb5-45de-9dff-56340eaf4b8a)
It works for me.

Reply:
Sorry, I commented on the wrong line here! My comment applies to className in `header`, and still applies.

**Some important notes:**
The root `columns.className` now appends the class name to `<tr><td>`. It should also append that class name to `<tr><th>`. That way the whole column, including header, can by styled.

If they also use `settings.header.className` that one should REPLACE the one from `columns.className`, if set.

If they use `settings.header.className` and `columns.className` is NOT set we should only append the class name to `<tr><th>`, not `<tr><td>`.

Reply:
> I suggest applying both of them. It will be more flexible and transparent for the end user. More classes does not have influence on performance or any other things.

I agree. Let's do it.

Reply:
> **3. If they use settings.header.className and columns.className is NOT set we should only append the class name to `th` , not `td`.**
> 
> Done

I can't see that this is fixed? No classname is appended to the `th`.

Reply:
We had a lot of merges / pulls yesterday, so was lost during it. I reverted the fix for applying proper classes.